### PR TITLE
Feature/release notes 9.23.0 custom dossier header modifiers

### PR DESCRIPTION
**Pull request by PeterK-Ritense**

### CODE REVIEW

**Review by mritense — APPROVED** [review]

**Comment by marijnritense** [comment]
Please only merge this PR once the other PR has been merged in (see https://github.com/valtimo-platform/valtimo-documentation/pull/126)

**Pull request merged**
